frontend/src/components/Profile/ProfileSignature.js

```diff
@@ -0,0 +1,30 @@
+import {Button} from "antd";
+import { Empty } from 'antd';
+import { Card } from 'antd';
+import {Context} from "../../store";
+import {useContext} from "react";
+
+
+const ProfileAbout = ({signature, username}) => {
+
+    const [state, dispatch] = useContext(Context);
+
+    return (
+        <Card title="Signature" hoverable>
+        {
+            signature
+            ?
+                <p>{signature}</p>
+            :
+            <Empty image={Empty.PRESENTED_IMAGE_SIMPLE} description={"This user has no signature"}/>
+        }
+            {
+                (state.auth.user.roles.includes('ROLE_ADMIN') || state.auth.user.username === username)
```

Comment:
Same question about admin updating someone else's signature - do we think this is OK?

Reply:
Admins have complete control, yup!